pom.xml

```diff
@@ -42,6 +42,14 @@
                         <configuration>
                             <mainClass>${main.class}</mainClass>
                             <imageName>bfsc-${os.detected.name}_${os.detected.arch}</imageName>
+                            <buildArgs>
+                                <!-- Optimize for compatibility -->
+                                <buildArg>-march=compatibility</buildArg>
+                                <!-- Optimize for size rather than speed -->
+                                <buildArg>-O1</buildArg>
+                                <!-- Build a static library with the exception of LibC -->
+                                <buildArg>--static-nolibc</buildArg>
```

Comment:
Build fails with `Unrecognized option: --static-nolibc`. Maybe we need to use a newer GraalVM instead of 17?

Reply:
Removed option. I will try the binaries under Ubuntu and Fedora. If they run there, that should be fine.